site/src/request_handlers/github.rs

```diff
@@ -158,6 +158,11 @@ async fn handle_rust_timer(
             Err(error) => errors.push_str(&format!("Cannot parse build command: {error}\n")),
         }
     }
+
+    if valid_build_cmds.is_empty() {
+        errors.push_str("@rust-timer command cannot be empty\n");
```

Comment:
Oops, there's a bigger problem with this. Because the bot used `@rust-timer` here, it triggers an endless sequence of bot comments 😆 https://github.com/rust-lang/rust/pull/112049#issuecomment-3542567996